src/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+WORKDIR /usr/src/app
+
+COPY . .
```

Comment:
Ensure that the context of the build does not include unnecessary files. Consider using a .dockerignore file to exclude files that are not needed in the Docker image.